README.md

```diff
@@ -78,6 +78,7 @@ This au3WebDriver UDF (project) allows to interact with any browser that support
 | _WD_ElementOptionSelect | Find and click on an option from a Select element.                              |
 | _WD_ElementSelectAction | Perform action on designated Select element.                                    |
 | _WD_ElementStyle        | Set/Get element style property.                                                 |
+| _WD_FindElementFrames   | Search the current document (including frames) and return locations of matching elements    |
```

Comment:
should we also add additionall spaces to reposiotion `|` for other functions ?

Reply:
Sure. FWIW, I've used [this site](https://www.tablesgenerator.com/markdown_tables) in the past to quickly layout tables in markdown format.

Reply:
Please take a look on: [Helper Functions - table indentation](https://github.com/Danp2/au3WebDriver/pull/386/commits/9c625a1350e862a8c55d005568c2e14c29598cbc)